docs/articles/vs-test-adapter/Adapter-Release-Notes.md

```diff
@@ -1,5 +1,25 @@
 # Adapter Release Notes
 
+## NUnit3 Test Adapter for Visual Studio - Version 4.0.0 - June 6, 2021
+
+This major release contains a series of changes, and also underlying changes in the adapter.  
+
+This is currently the version that will support Visual Studio 2022.
```

Comment:
Unclear whether you mean there are plans to backport support into prior versions, or whether the intention is that folks will have to upgrade to 4.x in order to use it with Visual Studio 2022.

Reply:
Updated the paragraph to make it more clear.  IF the VS team can manage to fix whatever is causing this in the versions they release, we will be fine.  Anything we do will result in an upgrade, and then the 4.X versions are that upgrade.